src/pybind/mgr/telemetry/module.py

```diff
@@ -744,9 +745,12 @@ def handle_command(self, inbuf, command):
             return 0, '', ''
         elif command['prefix'] == 'telemetry off':
             self.set_module_option('enabled', False)
-            self.set_module_option('last_opt_revision', REVISION)
+            self.set_module_option('last_opt_revision', 1)
             return 0, '', ''
         elif command['prefix'] == 'telemetry send':
+            if self.last_opt_revision < LAST_REVISION_RE_OPT_IN and command.get('license') != LICENSE:
+                self.log.debug('A telemetry send attempt while opted-out. Asking for license agreement')
+                return -errno.EPERM, '', "Telemetry data is licensed under the " + LICENSE_NAME + " (" + LICENSE_URL + ").\nTo manually send telemetry data, add '--license " + LICENSE + "' to the 'ceph telemetry send' command.\nPlease consider enabling the telemetry module with 'ceph telemetry on'."
```

Comment:
Just a note here: this argument works positionally too, e.g., ``ceph telemetry send thelicense``.  It might be less awkward (slightly less typing?) to have the use do that.

Reply:
Oh... let's leave this as is so they match then.